R/find.tol.time.R

```diff
@@ -1,3 +1,27 @@
+#' An internal function that find elution time tolerance level. 
+#' 
+#' This function finds the time tolerance level. Also, it returns the grouping information given the time tolerance.
+#' 
+#' @param mz mz value of all peaks in all profiles in the study.
+#' @param chr retention time of all peaks in all profiles in the study.
+#' @param lab label of all peaks in all profiles in the study.
```

Comment:
The label is sometimes used for the retention time - which is actually which?

Seems like the label is actually the rt group index.